### PR TITLE
[MM-45736] - Pricing Modal for Grandfathered Starter shows Free Starter Limits CTA

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR ensures that StarterDisclaimerCTA does not show for legacy starter product.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-45736

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by AGMETEOR** [comment]
> Cws@10102020

I have updated this spinwick to legacy cloud starter

**Comment by AGMETEOR** [comment]
@johndavidlugtu adding you so that you can have a look at the legacy starter feature list in the console

**Comment by johndavidlugtu** [comment]
@AGMETEOR thanks for tagging! The legacy Cloud Starter features in the System Console actually doesn't need to be updated. In the Jira ticket I mentioned that the proposed bullet points for the legacy Cloud Starter in the pricing modal was from the feature list from the System Console.

File: components/admin_console/billing/plan_details/feature_list.tsx [hunk 1]
```diff
@@ -115,10 +115,6 @@ const FeatureList = (props: FeatureListProps) => {
             id: 'admin.billing.subscription.planDetails.features.mfa',
             defaultMessage: 'Multi-Factor Authentication (MFA)',
         }),
-        intl.formatMessage({
-            id: 'admin.billing.subscription.planDetails.features.multilanguage',
-            defaultMessage: 'Multi-language translations',
-        }),
```
Comment (marianunez) [inline components/admin_console/billing/plan_details/feature_list.tsx:121]: Is this removal intended? This still applies to all Starter plans and the System Console card wouldn't require any changes

**Review by marianunez — CHANGES_REQUESTED** [review]
Condition to hide the limit disclaimer LGTM. However, we still need to change the pricing modal text for when they have no limits - see details in the ticket

**Comment by neallred** [comment]
Ditto what Maria said. Looks like that code lives in `components/pricing_modal/content.tsx`. We could check it by checking for a starter sku with no limits.

**Comment by AGMETEOR** [comment]
> @AGMETEOR thanks for tagging! The legacy Cloud Starter features in the System Console actually doesn't need to be updated. In the Jira ticket I mentioned that the proposed bullet points for the legacy Cloud Starter in the pricing modal was from the feature list from the System Console.

Updated
Screenshot 2022-07-21 at 19 15 00

**Review by marianunez — APPROVED** [review]
LGTM, thanks @AGMETEOR!

**Comment by marianunez** [comment]
> > @AGMETEOR thanks for tagging! The legacy Cloud Starter features in the System Console actually doesn't need to be updated. In the Jira ticket I mentioned that the proposed bullet points for the legacy Cloud Starter in the pricing modal was from the feature list from the System Console.
> 
> Updated Screenshot 2022-07-21 at 19 20 05

@AGMETEOR From the image, it seems legacy lost the `Current Plan` tag?

**Comment by AGMETEOR** [comment]
> > > @AGMETEOR thanks for tagging! The legacy Cloud Starter features in the System Console actually doesn't need to be updated. In the Jira ticket I mentioned that the proposed bullet points for the legacy Cloud Starter in the pricing modal was from the feature list from the System Console.
> > 
> > 
> > Updated Screenshot 2022-07-21 at 19 20 05
> 
> @AGMETEOR From the image, it seems legacy lost the `Current Plan` tag?

No this image is from local environment where my set up is not proper (i don't have products pulled in). I just wanted to use this image to show the copy. For a true image use the spinwick

**Review by neallred — APPROVED** [review]
LGTM, thanks @AGMETEOR !

**Review by johndavidlugtu — APPROVED** [review]
Good stuff @AGMETEOR !

**Comment by marianunez** [comment]
@stevemudie can we prioritize this PR for today? Would like to get this fix in for the upcoming cloud release

**Comment by stevemudie** [comment]
Looks good @AGMETEOR Just waiting on the E2E to run then I'll approve it.

**Comment by AGMETEOR** [comment]
> /cherry-pick cloud

Let me do it manually

**Comment by marianunez** [comment]
> > /cherry-pick cloud
> 
> Let me do it manually

Thanks @AGMETEOR

**Comment by AGMETEOR** [comment]
> > > /cherry-pick cloud
> > 
> > 
> > Let me do it manually
> 
> Thanks @AGMETEOR

https://github.com/mattermost/mattermost-webapp/pull/10831

**Pull request merged**